srcpkgs/cdk/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'cdk'
+pkgname=cdk
+version=5.0
+revision=1
+_date=20200228
```

Comment:
Why not make the data part of the version?

Reply:
Is it a rule?

Reply:
No, it just saves from having to update both revision and _date for every bump. 

That said, I don't like that it doesn't actually version releases.

Reply:
I've changed version to: `version=5.0-20200228`

> ERROR: version contains invalid character: -

Reply:
I was thinking more like `version=5.0.20200228` and `wrksrc="${pkgname}-${version%.*}-${version##*.}"`. Otherwise version will always be 5.0 and revision will just keep counting.

Reply:
Oh! `wrksrc` now is too complicated! Isn't it?

Reply:
Another possibility is `_majmin=5.0` `_date=20200228` and `version="${_majmin}.${_date}"` `wrksrc="${_majmin}-${_date}"`

Reply:
Please change to `wrksrc="${pkgname}-${version%.*}-${version##*.}"`, this works with https://github.com/leahneukirchen/xtools/issues/17 - like scripts.